app/uk/gov/hmrc/hec/services/scheduleService/HecTaxCheckExtractionService.scala

```diff
@@ -122,30 +122,38 @@ class HecTaxCheckExtractionServiceImpl @Inject() (
       count: Int,
       seqNumInt: Int,
       partialFileName: String,
-      dirname: String
+      dirname: String,
+      acc: List[EitherT[Future, Error, Unit]]
     ): List[EitherT[Future, Error, Unit]] = {
-      val hecList       = hecTaxCheckList.take(count)
-      val remainingList = hecTaxCheckList.drop(count)
-      val isRemaining   = remainingList.size > 0
+      val hecList          = hecTaxCheckList.take(count)
+      val remainingList    = hecTaxCheckList.drop(count)
+      val isLastInSequence = remainingList.size > 0
```

Comment:
or maybe we've got different ideas on what `isLastInSequence` means - I would have thought it was `true` if there was nothing else remaining

Reply:
[updated](https://github.com/hmrc/hec/commit/6384fb8a4f3fb21c3ce3e0575f4694633e1f31f9)